packages/search-ui-app-search-connector/src/__tests__/AppSearchAPIConnector.test.js

```diff
@@ -193,7 +193,8 @@ describe("AppSearchAPIConnector", () => {
           {
             field: "world_heritage_site",
             values: ["true"],
-            type: "all"
+            outerType: "all",
```

Comment:
By changing this test case to have `outerType: "all"`, it's hard to tell if anything has broken or not. Previously, omitting `outerType` would default the outer type to "all". Does it still? I can't tell in this test. I think I would actually just remove it from this test and just test the `outerType` behavior in `requestAdapters.test.js`.